src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -109,7 +109,8 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
   cleanup(): void {
     if (this.mediaStream) {
       for (let i = 0; i < this.mediaStream.getTracks().length; i++) {
-        this.mediaStream.getTracks()[i].stop();
+        const track = this.mediaStream.getTracks()[i];
```

Comment:
is this just coding style change?

Reply:
It's not a semantic change, correct.

I made the change in order to add logging (later removed), but I do find it slightly more readable — it splits the `()[].` sequence — and the extra line adds an easy breakpoint for those who use debuggers.